### PR TITLE
Add UploadStoreSalesTransactions example

### PR DESCRIPTION
**Pull request by jradcliff**
Change-Id: I0bfb7fa54d96138276296121609bb17d8172adf3

### CODE REVIEW

**Comment by jradcliff** [comment]
See internal review at [tg/751712](http://tg/751712).

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/UploadStoreSalesTransactions.java [hunk 1]
```diff
@@ -0,0 +1,505 @@
...
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+/**
+ * Uploads offline data for store sales transactions.
+ *
+ * <p>This feature is only available to whitelisted accounts. See
```
Comment (sundquist) [inline google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/UploadStoreSalesTransactions.java:58]: Can we change whitelisted to allowlisted? I think right now we're attempting to change our wording a bit.

Comment (nwbirnie) [reply google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/UploadStoreSalesTransactions.java:58]: +1

Comment (jradcliff) [reply google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/UploadStoreSalesTransactions.java]: Good point. Fixed.

**Pull request merged**